client/asset/dcr/config.go

```diff
@@ -31,17 +31,18 @@ var (
 // Config holds the parameters needed to initialize an RPC connection to a dcr
 // wallet. Default values are used for RPCListen and/or RPCCert if not set.
 type Config struct {
-	PrimaryAccount   string  `ini:"account"`
-	UnmixedAccount   string  `ini:"unmixedaccount"`
-	TradingAccount   string  `ini:"tradingaccount"`
-	RPCUser          string  `ini:"username"`
-	RPCPass          string  `ini:"password"`
-	RPCListen        string  `ini:"rpclisten"`
-	RPCCert          string  `ini:"rpccert"`
-	UseSplitTx       bool    `ini:"txsplit"`
-	FallbackFeeRate  float64 `ini:"fallbackfee"`
-	FeeRateLimit     float64 `ini:"feeratelimit"`
-	RedeemConfTarget uint64  `ini:"redeemconftarget"`
+	PrimaryAccount          string  `ini:"account"`
+	UnmixedAccount          string  `ini:"unmixedaccount"`
+	TradingAccount          string  `ini:"tradingaccount"`
+	RPCUser                 string  `ini:"username"`
+	RPCPass                 string  `ini:"password"`
+	RPCListen               string  `ini:"rpclisten"`
+	RPCCert                 string  `ini:"rpccert"`
+	UseSplitTx              bool    `ini:"txsplit"`
+	FallbackFeeRate         float64 `ini:"fallbackfee"`
+	FeeRateLimit            float64 `ini:"feeratelimit"`
+	RedeemConfTarget        uint64  `ini:"redeemconftarget"`
+	AllowExternalAPIRequest bool    `ini:"allowexternalapirequest"`
```

Comment:
About half of this diff is just because you picked a long name here and in `ExchangeWallet`. Can we not find a shorter way to say this? How about `ApiFeeFallback`? 

`AllowExternalAPIRequest` doesn't mention fees anyway.